hack/verify-lint-config.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright 2017 the Velero contributors.
+# Copyright 2020 the Velero contributors.
```

Comment:
The current dev guidelines call for omitting the year in the Copyright notice.

Reply:
done